ERCS/erc-draft-data-callback.md

```diff
@@ -0,0 +1,233 @@
+---
+eip: XXXX
```

Comment:
```suggestion
eip: 8026
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.